api/v1/serializers/instance_serializer.py

```diff
@@ -51,6 +52,14 @@ def __init__(self, *args, **kwargs):
         self.request_user = user
         super(InstanceSerializer, self).__init__(*args, **kwargs)
 
+    def get_size(self, obj):
+        from api.v2.serializers.summaries import (
```

Comment:
@kamfonik hmm, I will ask you on this then. Since this is only used by this function. Sometimes I see people import them within only the functions. But we could also put them on top. What do you think, Laura?